dev-tools/build-packages/base/builder.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Wazuh dashboard base builder
+# Copyright (C) 2021, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+set -e
+
+# Script parameters to build the package
+architecture="$1"
+revision="$2"
+repository="$3"
+version="$4"
+base_dir=/opt/wazuh-dashboard-base
+
+# -----------------------------------------------------------------------------
+
+if [ -z "${revision}" ]; then
+    revision="1"
+fi
+
+if [ "${architecture}" = "x86_64" ] || [ "${architecture}" = "amd64" ]; then
+    architecture="x64"
+fi
+
+wazuh_minor=$(echo ${version} | cut -c1-3)
+
+# Obtain Wazuh plugin URL
+if [ "${repository}" ];then
+    valid_url='(https?|ftp|file)://[-[:alnum:]\+&@#/%?=~_|!:,.;]*[-[:alnum:]\+&@#/%=~_|]'
```

Comment:
I was trying to make the packages with local files and when I run the `./generate_base.sh` file it is not working, and I get this error: 
`The given URL to download the Wazuh plugin zip does not exist: file:///home/ian/wazuh/Wazuh/Wazuh-dashboard/wazuh-dashboard/target/wazuh-4.5.0-1.zip`

Although in the code from what I understand it would accept the URL to be like this